agenthub/codeact_agent/codeact_agent.py

```diff
@@ -22,6 +28,8 @@
     SWEAgentCommandsRequirement,
 )
 
+ENABLE_GITHUB = True
```

Comment:
I am actually thinking if we can maintain a library of pre-defined prompts (skill) where user can select on the frontend - e.g., they can click the button to select [Github prompt] and automatically load the content into their input box (as the first user message), then they append their requirement to that prompt.